administrator/components/com_media/resources/scripts/store/mutation-types.es6.js

```diff
@@ -48,5 +48,9 @@ export const RENAME_SUCCESS = 'RENAME_SUCCESS';
 // Share model
```

Comment:
```suggestion
// Share modal
```
I know it is unrelated to this PR, but it is not worth opening a separate PR for an obvious typo. Thanks.